app/components/SearchBox.tsx

```diff
@@ -34,24 +34,23 @@ const SearchBox: React.FunctionComponent<Props> = ({
     type: 'object',
     properties: {
       searchField: {
+        title: 'Search By:',
         type: 'string',
         enum: dropdownSortItems
       },
       searchValue: {
+        title: 'Search Term:',
         type: 'string'
       }
     }
   };
 
   const uiSchema = {
     searchField: {
-      'ui:placeholder': 'Search By: ',
-      'ui:col-md': 5,
-      classNames: 'hide-title'
+      'ui:col-md': 5
     },
     searchValue: {
-      'ui:col-md': 7,
-      classNames: 'hide-title'
```

Comment:
The `.hide-title` class would hide the [`<label>`s required to be set above](https://github.com/bcgov/cas-ciip-portal/pull/1013/files#diff-015f9994a81656a7479a8042902172d0R37) using `display: none;`. 

(Visual hiding of DOM objects using CSS also hides those elements from screenreaders.)